x/ibc-hooks/README.md

```diff
@@ -180,6 +180,104 @@ pub enum SudoMsg {
 }
 ```
 
+### Async Acks
+
+IBC supports the ability to send an ack back to the sender of the packet asynchronously. This is useful for
+cases where the packet is received, but the ack is not immediately known. For example, if the packet is being
+forwarded to another chain, the ack may not be known until the packet is received on the other chain.
+
+#### Making contract Acks async
```

Comment:
Could you please add a section similar to "Use case" under "Ack callbacks" and compare async vs non-sync? I think having that comparison on a real example would be useful